testsuite/python/mdanalysis.py

```diff
@@ -24,10 +24,11 @@
 import espressomd.interactions
 import numpy as np
 import unittest as ut
+import unittest_decorators as utx
 try:
     import MDAnalysis as mda
     from espressomd import MDA_ESP
-    skipIfMissingPythonPackage = ut.case._id
+    skipIfMissingPythonPackage = utx._id
```

Comment:
```suggestion
    skipIfMissingPythonPackage = lambda x: x
```

Reply:
this would be even more readable, right?

Reply:
we want to hide implementation details of python decorators (that's why `unittest` created `_id()` in the first place)

also, that lambda expression would be transformed into a regular function by autopep8